demo/index.js

```diff
@@ -345,6 +348,15 @@ api.on('api:error', function(data) {
     console.log('Error :(', data.resourceId);
 });
 
+app.use("/docs", chain(cors(), apiDocs(api, {
```

Comment:
Can this be moved into the framework so we don't have to write a custom endpoint to generate documentation? See the `respondDocs` function in Router.js.